restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/telephony/DialActionOrganizationSBCTest.java

```diff
@@ -0,0 +1,1079 @@
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+import org.mobicents.ext.javax.sip.dns.DNSLookupPerformer;
+import org.mobicents.ext.javax.sip.dns.DefaultDNSLookupPerformer;
+import org.restcomm.connect.commons.Version;
+import org.restcomm.connect.commons.annotations.SequentialClassTests;
+import org.restcomm.connect.commons.annotations.WithInMinsTests;
+import org.restcomm.connect.testsuite.http.RestcommCallsTool;
+import org.restcomm.connect.testsuite.http.RestcommConferenceParticipantsTool;
+import org.restcomm.connect.testsuite.http.RestcommConferenceTool;
+import org.restcomm.connect.testsuite.tools.MonitoringServiceTool;
+import org.xbill.DNS.DClass;
+import org.xbill.DNS.NAPTRRecord;
+import org.xbill.DNS.Name;
+import org.xbill.DNS.Record;
+import org.xbill.DNS.SRVRecord;
+import org.xbill.DNS.TextParseException;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.github.tomakehurst.wiremock.verification.LoggedRequest;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+
+import gov.nist.javax.sip.stack.HopImpl;
+
+/**
+ * Test for Dial Action attribute for organization
+ *
+ * @author maria
+ *
+ */
+@RunWith(Arquillian.class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+@Category(value={WithInMinsTests.class, SequentialClassTests.class})
+public class DialActionOrganizationSBCTest {
+
+    private final static Logger logger = Logger.getLogger(DialActionOrganizationSBCTest.class.getName());
+    private static final String TRANSPORT = "udp";
+
+    private static final String version = Version.getVersion();
+    private static final byte[] bytes = new byte[] { 118, 61, 48, 13, 10, 111, 61, 117, 115, 101, 114, 49, 32, 53, 51, 54, 53,
+        53, 55, 54, 53, 32, 50, 51, 53, 51, 54, 56, 55, 54, 51, 55, 32, 73, 78, 32, 73, 80, 52, 32, 49, 50, 55, 46, 48, 46,
+        48, 46, 49, 13, 10, 115, 61, 45, 13, 10, 99, 61, 73, 78, 32, 73, 80, 52, 32, 49, 50, 55, 46, 48, 46, 48, 46, 49,
+        13, 10, 116, 61, 48, 32, 48, 13, 10, 109, 61, 97, 117, 100, 105, 111, 32, 54, 48, 48, 48, 32, 82, 84, 80, 47, 65,
+        86, 80, 32, 48, 13, 10, 97, 61, 114, 116, 112, 109, 97, 112, 58, 48, 32, 80, 67, 77, 85, 47, 56, 48, 48, 48, 13, 10 };
+    private static final String body = new String(bytes);
+
+    @ArquillianResource
+    private Deployer deployer;
+    @ArquillianResource
+    URL deploymentUrl;
+
+	@GetDeployableContainer
+	private ContainerManagerTool containerManager;
+
+    @Rule
+    public WireMockRule wireMockRule = new WireMockRule(8090); // No-args constructor defaults to port 8080
```

Comment:
this is not following paralle testing approach, why dont we use paralel testing here?

Reply:
I copied the DialActionOrganizationTest.java, not sure if it was present initially there. How would you convert that to parallel testing ?